app/models/full_refresh_upload_task.rb

```diff
@@ -1,2 +1,14 @@
 class FullRefreshUploadTask < MqttClientTask
+  after_update :post_full_refresh_upload_task, :if => proc { type == 'FullRefreshUploadTask' && state == 'completed' }
+
+  def post_full_refresh_upload_task
+    PostRefreshUploadTaskService.new(service_options).process
+  end
+
+  def service_options
```

Comment:
@hsong-rh This can be used from the parent we dont need it here. The SourceID and TenantID fields can be pulled from the task object.